scripts/etc/init.d/rcS

```diff
@@ -1284,6 +1284,17 @@ if [ "$kernel_module" = true ] ; then
   fi
 fi
 
+# install firmware for wireless chipset (RPi 3 and Zero W)
+if [ "$ifname" != "eth0" ]; then
+  echo -n "Installing firmware for Broadcom wireless network cards... "
+  chroot /rootfs /usr/bin/apt-get -y install firmware-brcm80211 &>/dev/null
+  if [ $? -eq 0 ]; then
+    echo "OK"
+  else
+    echo "FAILED !"
+  fi
+fi
+
```

Comment:
My concern here: Please look at lines 335-346, 605-612 & 1120-1127 -- which (also) supports WiFi dongles -- and confirm that these pieces of code will work together with your change. 
I have no possibility to test this anymore :-(